setup.py

```diff
@@ -29,58 +29,39 @@ def get_version():
     raise RuntimeError('Unable to find version string.')
 
 
-class Tox(TestCommand):
-    user_options = [('tox-args=', 'a', 'Arguments to pass to tox')]
-
-    def initialize_options(self):
-        TestCommand.initialize_options(self)
-        self.tox_args = None
-
-    def finalize_options(self):
-        TestCommand.finalize_options(self)
-        self.test_args = []
-        self.test_suite = True
-
-    def run_tests(self):
-        # Import here. Outside the .eggs/ will not load
-        import tox
-        import shlex
-        args = self.tox_args
-        if args:
-            args = shlex.split(self.tox_args)
-        errno = tox.cmdline(args=args)
-        sys.exit(errno)
-
 setup(
     name='jwplatform',
     version=get_version(),
     description='A Python client library for accessing JW Platform API',
     long_description=read_file('README.rst') + '\n\n' + read_file('CHANGES.rst'),
     url='https://github.com/jwplayer/jwplatform-py',
-    author='Sergey Lashin, LongTail Ad Solutions, Inc.',
+    author='Kamil Sindi',
     author_email='support@jwplayer.com',
     license='MIT',
     classifiers=[
         'Development Status :: 5 - Production/Stable',
         'Intended Audience :: Developers',
         'Topic :: Software Development :: Libraries :: Python Modules',
         'License :: OSI Approved :: MIT License',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
     ],
```

Comment:
Should we add support for 3.8 too?

Reply:
Makes sense. I'll also note it as Python 3 only.